openmdao/docs/build_jupyter_book.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import argparse
+import os
+import pathlib
+import fnmatch
+import shutil
+
+
+_this_file = pathlib.Path(__file__).resolve()
+REPO_ROOT = _this_file.parent
+BOOK_DIR = pathlib.Path(REPO_ROOT, 'openmdao_book')
+
+
+def copy_build_artifacts(book_dir=BOOK_DIR):
```

Comment:
it looks like this function is duplicated in a  standalone file

Reply:
Fixed